ann_benchmarks/runner.py

```diff
@@ -137,13 +148,18 @@ def run(definition, dataset, count, run_count, batch):
             if query_arguments:
                 algo.set_query_arguments(*query_arguments)
             descriptor, results = run_individual_query(
-                algo, X_train, X_test, distance, count, run_count, batch)
+                algo, X_train, X_test, distance, count, run_count, batch, batch_size)
             descriptor["build_time"] = build_time
             descriptor["index_size"] = index_size
             descriptor["algo"] = definition.algorithm
             descriptor["dataset"] = dataset
+
+            print(str(descriptor))
             store_results(dataset, count, definition,
                           query_arguments, descriptor, results, batch)
+    except:
+        print("FAILED!")
+        print(str(e))
```

Comment:
Since this seems to have been important in your testing, maybe make this prettier but keep it?